### ISSUE TITLE
Export Report to PDF file (Table, Report etc.) consistently shows 16 accessibility failures with checked using Adobe Accessibility Checker (504.2.2 PDF Export)

### ISSUE THREAD

**Issue by msftedad** [adobe/adobe.github.com#141]
### Expected Behaviour
PDF file should not have any accessibility issue after converting it from Access.

### Actual Behaviour
Adobe Accessibility Checker shows 16 issues in PDF file.

#### Steps to Reproduce
Step 1: Open Access application.
Step 2: Open Northwind document.
Step 3: Get the focus on ribbon using Ctrl + F6 key.
Step 4: Navigate on External Data Tab and activate it using keyboard.
Step 5: Navigate on PDF or XPS and activate it using keyboard.
Step 6: Export the file into PDF.
Step 7: Open Adobe Acrobat and navigate on accessibility check and activate it.
Step 8: Verify the accessibility issue on PDF file and observe the issue.

#### Platform and Version
Adobe Version: 2023
#### Sample Code that illustrates the problem
[frmMainMenu.pdf](https://github.com/adobe/adobe.github.com/files/13873589/frmMainMenu.pdf)

#### Logs taken while reproducing problem
![Access to PDF-Accessibility Issue 2](https://github.com/adobe/adobe.github.com/assets/93735775/7df5d523-e615-4a6d-9d79-87c3c7867823)
![Access to PDF-Accessibility Issue 1](https://github.com/adobe/adobe.github.com/assets/93735775/fcc7c695-95fb-4162-94c8-8215f6f9903c)

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Is there any update on this issue?
